### ISSUE TITLE
Faulty trigger events.

### ISSUE THREAD

**Issue by NiktosNOTD** [ArcanePariah/Night-of-the-Dead#240]
When i was optimizing events i made a mistake of sorting them by "Failed Conditions" and as result of it i missed one super spammy event. It's a recruit mode game, in -nm it for sure runs even more times.
![bosshppls](https://user-images.githubusercontent.com/30372657/40498683-f3035522-5f7f-11e8-969b-7df502874c90.png)
[boss hp event example.zip](https://github.com/ArcanePariah/Night-of-the-Dead/files/2036256/boss.hp.event.example.zip)

Yes it's 418 thousands of occurences. It's native blizzard made event for updating boss hp bars, but it got an error within event and fires like mad for everything. As also seen in this screenshot it was warranted for less than 5 thousand times, with rest being needless.

Given it's a native function it can't be directly fixed just replaced with custom one. 
I guess system from team-sidebar which keeps track of marines hp could be adapted to it.

**Comment by dave-sWpR46Ae** [comment]
This can be fixed with custom data based HP bars.  CP2 just did this migration, so might look into asking folk about it.

**Comment by ArcanePariah** [comment]
Agreed, and for some reason, I think I did precisely this in NOTD 2. So we may just backport it. I can inquire with Folk as well, I chat with him on mapster discord.

But overall yes, we can create our own custom "boss bar" and use it, and I can add a bit of code to destroy or halt the usage of the Blizzard library code.
